SU2_CFD/include/fluid/CFluidModel.hpp

```diff
@@ -122,29 +130,12 @@ class CFluidModel {
   /*!
    * \brief Get fluid specific heat at constant pressure.
    */
-  su2double GetCp() const { return Cp; }
+  inline virtual su2double GetCp() const { return Cp; }
```

Comment:
Why are these virtual now? Please avoid virtual functions.

Reply:
Yes, they should not be virtual functions. Thanks for the comment!!